start

```diff
@@ -1,6 +1,12 @@
 #! /usr/bin/env bash
 set -e
 
+clear=$(echo -e "\e[0m")
```

Comment:
aha, colors are back! On previous trial with these, I did notice that some of these codes don't always equate to the same on screen colors, probably dependent on theme settings of host system, etc, whatever color realized, the prefix on each line keeps it clear what belongs to what.

Reply:
Yup, the main benefit is to distinguish the lines a little clearer. The variables are named after their standard ANSI colors, however anyone can redefine the colors in their terminal program.

We should use colors very sparingly. When I mentioned colors previously this was the use case I was referring to: using it as a way to help separate large amounts of logs.

Reply:
The format is directly based on the format of `docker-compose` and `foreman` outputs.